core/store/migrations/migration1604437959/migrate.go

```diff
@@ -0,0 +1,27 @@
+package migration1604437959
+
+import "github.com/jinzhu/gorm"
+
+const up = `
+	CREATE TABLE "pipeline_spec_errors" (
+		"id" bigserial primary key,
+		"pipeline_spec_id" int REFERENCES pipeline_specs (id),
+		"description" text NOT NULL,
+		"occurrences" integer DEFAULT 1 NOT NULL,
+		"created_at" timestamptz NOT NULL,
+		"updated_at" timestamptz NOT NULL
+	);
+	CREATE UNIQUE INDEX pipeline_spec_errors_unique_idx ON pipeline_spec_errors ("pipeline_spec_id", "description");
```

Comment:
Let's add BRIN indexes on created_at and updated_at.

Reply:
Might this be a bit premature? Are we expecting to have lots of unique spec errors?

Reply:
Yeah without knowing anything about the load on this table yet, I think it is premature